data/json/items/tool/cooking.json

```
@@ -878,7 +878,7 @@
     "id": "still",
     "type": "TOOL",
     "name": { "str": "still" },
-    "description": "This is a still.  It's useful for creating distillation of alcohol and other, more curious substances.",
+    "description": "This is a still.  It's useful for creating distillation of alcohol and other, more curious substances. Features an in-build slot to insert thermometer into.",
```

```suggestion
    "description": "This is a still.  It's useful for distilling alcohol and other, more curious substances. 
 Features a built-in slot for a thermometer.",
```
As well as the spacing issue ZhilkinSerg pointed out.